NineChronicles.Headless.AccessControlCenter/AccessControlService/MutableRedisAccessControlService.cs

```diff
@@ -18,10 +18,24 @@ public MutableRedisAccessControlService(string storageUri)
 
         public void DenyAccess(Address address)
         {
-            _db.StringSet(address.ToString(), "denied");
+            _db.StringSet(address.ToString(), "0");
         }
 
         public void AllowAccess(Address address)
+        {
+            var value = _db.StringGet(address.ToString());
+            if (value == "0")
+            {
+                _db.KeyDelete(address.ToString());
+            }
```

Comment:
It seems to use the same key for several purposes. I think it is an easy way to cause bugs. How about using different keys with prefixes? If it removes tx quota, it will be `access-denined` state.